### DISCUSSION TITLE
I looking for a cmdlet that would help me update existing content type document template url.

### DISCUSSION THREAD

**Discussion by dioulos** [pnp/powershell#3061] [Q&A]
Hello,

I got some content type which have typo in their document template relative path. I would like to update the content type document template url. Is it possible with PNP?

Note that these content type are under the hidden group

**Comment by Rangnarok123** [comment]
Try this?

https://pnp.github.io/powershell/cmdlets/Set-PnPContentType.html
Works with Set/Get/Remove/Add
